packages/sdk/src/HopBridge.ts

```diff
@@ -745,33 +771,41 @@ class HopBridge extends Base {
   ): Promise<BigNumber> {
     sourceChain = this.toChainModel(sourceChain)
     destinationChain = this.toChainModel(destinationChain)
+    const token = this.toTokenModel(this.tokenSymbol)
+
     const [credit, debit] = await Promise.all([
       this.getCredit(destinationChain, bonder),
       this.getDebit(destinationChain, bonder)
     ])
 
     let availableLiquidity = credit.sub(debit)
 
-    if (
-      sourceChain.equals(Chain.Optimism) &&
-      destinationChain.equals(Chain.Ethereum)
-    ) {
-      const bridgeContract = await this.getBridgeContract(sourceChain)
-      const pendingAmount = await bridgeContract.pendingAmountForChainId(
-        destinationChain.chainId
-      )
-
-      availableLiquidity = availableLiquidity.sub(pendingAmount)
-
-      const token = this.toTokenModel(this.tokenSymbol)
-      if (token.symbol === 'USDC') {
-        const unbondedRootsBufferBn = parseUnits(
-          UnbondedRootsBuffer,
-          token.decimals
-        )
-        availableLiquidity = availableLiquidity.sub(unbondedRootsBufferBn)
+    let minusBuffer = false
+    if (destinationChain.equals(Chain.Ethereum)) {
+      if (
+        sourceChain.equals(Chain.Optimism) ||
+        sourceChain.equals(Chain.Arbitrum)
+      ) {
+        minusBuffer = true
       }
     }
+    if (minusBuffer) {
```

Comment:
I was under the impression we were removing this in favor of 2x. If so, the whole concept of `UnbondedRootsBufferAmount` should be removed.

Reply:
Fixed